src/hooks/useDailyStatistics/utils.ts

```diff
@@ -40,7 +42,13 @@ export const countTotalTransactionsAndByCategory = (
               type: "POSTFIX",
               label: " qty"
             }
-          )
+          ),
+          descriptionAlert:
+            policies?.find(item => item.category === key)?.categoryType ===
+            "APPEAL"
+              ? "via appeal"
+              : undefined,
+          order: policies?.find(item => item.category === key)?.order ?? -1
```

Comment:
Perhaps you can define `policies?.find(item => item.category === key)` in a `const` above and then refer to it's properties here, looks like you're repeating it a lot!

Reply:
good call, changed this to `findItemByCategory`!